eda_mibici.py

```diff
@@ -52,3 +52,18 @@
 
 df = pd.read_csv('datos.csv')
 df = df.rename(columns={"A�o_de_nacimiento": "Anio_de_nacimiento"})
+
+import numpy as np
+import datetime
+from dateutil.relativedelta import relativedelta
+from datetime import date
+ 
+#Sacamos la diferencia en segundos y se agrega a una columna llamada diff_seconds
+df['diff_seconds'] = df['Fin_del_viaje'] - df['Inicio_del_viaje']
+df['diff_seconds']= df['diff_seconds']/np.timedelta64(1,'s')
```

Comment:
```suggestion
df['diff_seconds'] = (df['Fin_del_viaje'] - df['Inicio_del_viaje']).astype('timedelta64[s]')
```

Me parece que desde un inicio podemos decirle de qué tipo queremos que sea la resta, sin necesidad de hacer la división.